JS-301: AJAX Basics /workspace/example1.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link
+      href="http://fonts.googleapis.com/css?family=Varela+Round"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="../css/style.css" />
+    <title>AJAX with JavaScript</title>
+    <script></script>
```

Comment:
Did you forget to supply `script` source file? Script should be placed before the closing `</body>`.